pyserini/search/nmslib/__init__.py

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 #
 
-from ._vsearcher import SearchResult, SimpleVectorSearcher
+from ._searcher import SearchResult, NmslibSearcher
 
-__all__ = ['SearchResult', 'SimpleVectorSearcher']
+__all__ = ['SearchResult', 'NmslibSearcher']
```

Comment:
`NmslibSearcher` is correct, but looks _ugly_!

Reply:
hrm... have to match Lucene and Faiss i guess